core/governance/engine.go

```diff
@@ -713,6 +723,12 @@ func (e *Engine) validateOpenProposal(proposal *types.Proposal) (types.ProposalE
 		}
 	}
 
+	// assign all requirement to the proposal itself.
```

Comment:
Minor: Do we need to do this both at the start and at the end of `validateOpenProposal`?

Reply:
not at all, only at the start. it's a mistake